android/app/src/main/kotlin/com/jabook/app/jabook/audio/AudioPlayerPostInitCoordinator.kt

```
@@ -0,0 +1,105 @@
+        initializeVisualizer()
+    }
+
+    private fun restorePlaybackSpeed() {
+        service.playerServiceScope.launch {
+            try {
+                val savedSpeed = service.audioPreferences.playbackSpeed.first()
+                withContext(Dispatchers.Main) {
+                    LogUtils.d("AudioPlayerService", "Restoring playback speed: ${savedSpeed}x")
+                    service.exoPlayer.setPlaybackSpeed(savedSpeed)
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)